app/src/main/java/com/example/poste/models/User.java

```diff
@@ -209,13 +209,29 @@ public void onResponse(@NonNull Call<ResponseBody> call, @NonNull Response<Respo
                                 String postDescription = post.getString("description");
                                 String postUrl = post.getString("url");
                                 String postId = post.getString("id");
+                                String tagList = "";
+                                if (post.has("tags")) {
+                                    JSONArray postTags = post.getJSONArray("tags");
+                                    for (int p = 0; p < postTags.length(); p++) {
+                                        tagList = tagList + postTags.getString(p);
+                                        if (p < postTags.length() - 1) {
+                                            tagList = tagList + ",";
```

Comment:
I don't think it is a good idea to conver the JSON Array into a String, and then pass that string into the setTags method to convert it back into a list.
You cna try the following
https://stackoverflow.com/questions/3395729/convert-json-array-to-normal-java-list

Reply:
Changed. We now just go strait from json array to arraylist which is given to the builder.